app/presenters/online_booking_onboarding_banner.rb

```diff
@@ -0,0 +1,15 @@
+class OnlineBookingOnboardingBanner
+  def initialize(organisation)
+    @organisation = organisation
+  end
+
+  def display?
+    motifs_with_missing_availabilities.any?
+  end
+
+  def motifs_with_missing_availabilities
+    @motifs_with_missing_availabilities ||= @organisation.motifs.bookable_by_everyone.select do |motif|
```

Comment:
```suggestion
    @motifs_with_missing_availabilities ||= @organisation.motifs.active.bookable_by_everyone.select do |motif|
```

En testant localement j'ai constaté qu'un motif archivé déclenchait l'affichage de la banière. 😉

Reply:
bien vu ! Merci pour le fix